src/sentry/api/endpoints/project_rule_enable.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import audit_log
+from sentry.api.api_owners import ApiOwner
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.endpoints.project_rules import find_duplicate_rule
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.constants import ObjectStatus
+from sentry.models import Rule
+
+
+@region_silo_endpoint
+class ProjectRuleEnableEndpoint(ProjectEndpoint):
```

Comment:
why create an endpoint instead of having this within the same endpoint where rules are disabled?

Reply:
This case is kinda weird - we're going to run a script to disable rules and let users fix them up if they want rather than just deleting stuff.